test/completion.js

```diff
@@ -38,4 +38,17 @@ describe('flag: --completion', function() {
     }
   });
 
+  it('shows error message for missing completion type', function(done) {
+    runner({ verbose: false })
+      .gulp('--completion')
+      .run(cb);
+
+    function cb(err) {
+      expect(function() {
+        throw err;
```

Comment:
why do we need to throw this error?  I think you can test the message with match or something.